widgets/BigQueryPushdownEngine-sqlengine.json

```diff
@@ -189,6 +189,22 @@
             },
             "default": "false"
           }
+        },
+        {
+          "widget-type": "toggle",
+          "label": "Attempt direct copy to BigQuery sinks",
+          "name": "useDirectSinkWrite",
+          "widget-attributes": {
+            "on": {
+              "value": "true",
+              "label": "YES"
+            },
+            "off": {
+              "value": "false",
+              "label": "NO"
+            },
+            "default": "false"
+          }
```

Comment:
should this be added to ```BigQueryPushdownEngine-sqlengine.md``` ?

Reply:
Good observation. I'll work on writing these docs with Robin.